packages/components/src/types/page.ts

```diff
@@ -39,14 +39,11 @@ const TagCategorySchema = Type.Composite([
   DropdownItemSchema,
 ])
 // NOTE: can be optional because the categories might not exist
-const TagCategoriesSchema = Type.Object(
-  {
-    tagCategories: Type.Optional(
-      Type.Array(TagCategorySchema, { format: "hidden" }),
-    ),
-  },
-  { format: "hidden" },
-)
+const TagCategoriesSchema = Type.Object({
+  tagCategories: Type.Optional(
+    Type.Array(TagCategorySchema, { format: "tag-categories" }),
```

Comment:
can make `tag-categories` a const? feel like easier to debug between components and studio

Reply:
@dcshzj had this idea that components **should only export schemas/types** - that's the contract between studio as a client and components as the server. to that end, we should not be exporting `const` and other values from components.

rather, these are _studio_ constructs that are derived from the schema. instead of doing the export from `components`, we should define a file in studio (or a folder or smth) that basically defines and exports all these `const` or default values since they are either artificial constructs or inferred from the schema. 

wdyt about this approach @adriangohjw ?